### DISCUSSION TITLE
Maybe no Need of  GS Transform node !

### DISCUSSION THREAD

**Discussion by FreeFlowVFX** [cgnomads/GSOPs#5] [Ideas]
I think you can use this : `setattribtypeinfo(0,"point","scale","vector");` to convert your scale attribute to vector . Then the regular houdini transform node would work fine .

**Comment by david-rhodes** [comment]
Tim, thank you so much for starting the discussion!

You've just blown my mind. 🤯 I searched for this functionality and was confused that it was not working by default. I'm surprised since scale is literally a vector attribute . . .

Anyway, thank you! This certainly improves the UX.

**Comment by david-rhodes** [comment]
Implemented: https://github.com/david-rhodes/GSOPs/commit/825853bd94479d6feb6ff5843618868de5345984. 

Thanks again @FreeFlowVFX!
